### ISSUE TITLE
Unable to gather followees

### ISSUE THREAD

**Issue by RandomOverflow** [Thammus/instaloader#19]
I try to get followers from a random instagram profile but the program always gives this error:
Fatal error: ConnectionError(400): unable to gather followees.

I used the following command: 
python instaloader.py --login=MY_USERNAME @TARGET_USERNAME

**Comment by RandomOverflow** [comment]
I got this error:
https://i.imgur.com/LZCaTX9.png
Anyway can you make possible to download all profile's followers in a TXT file? No photos, ids, just names
Thank you again

**Comment by Thammus** [comment]
Thank you for reporting the errors, they should have been fixed now with commits 3614455, 8e77a1c and 0e94318.
To clarify some terminology: By followees we mean profiles that are followed, not to be confused with followers. If you follow some people on Instagram, these are your followees. If someone is following you, then this is your follower.
So, do you want to download all profile's followers or followees into a TXT file? If you meant followees, then this could simply be done by using instaloader as a library in a small python script like this:

```python
import instaloader

your_username = 'jacopob43'
target_profile = 'badass_beards_and_beautyqueens'

session = instaloader.get_logged_in_session(your_username)

followees = instaloader.get_followees(target_profile, session)

with open(target_profile+"_followees.txt", "w") as textfile:
    for f in followees:
        _ = textfile.write(f['username']+"\n")
```

**Comment by Deleora** [comment]
> Thank you for reporting the errors, they should have been fixed now with commits [3614455](https://github.com/instaloader/instaloader/commit/361445519a3dc240aea35309d98b8d8271fdab33), [8e77a1c](https://github.com/instaloader/instaloader/commit/8e77a1c125aa738571d4574bb9a2ecfd08418cd2) and [0e94318](https://github.com/instaloader/instaloader/commit/0e943189e50106da5813be9df754db70257cf28e).
> To clarify some terminology: By followees we mean profiles that are followed, not to be confused with followers. If you follow some people on Instagram, these are your followees. If someone is following you, then this is your follower.
> So, do you want to download all followers or followees of a given profile into a TXT file? If you meant followees, then this could simply be done by using instaloader as a library in a small python script like this:
> 
> ```python
> import instaloader
> 
> your_username = 'jacopob43'
> target_profile = 'badass_beards_and_beautyqueens'
> 
> session = instaloader.get_logged_in_session(your_username)
> 
> followees = instaloader.get_followees(target_profile, session)
> 
> with open(target_profile+"_followees.txt", "w") as textfile:
>     for f in followees:
>         _ = textfile.write(f['username']+"\n")
> ```

How to download both 'followers' & 'following' of a profile?
